lms/templates/navigation-edx.html

```diff
@@ -51,28 +52,30 @@ <h1 class="logo" itemscope="" itemtype="http://schema.org/Organization">
     % endif
 
     % if user.is_authenticated():
-    <div class="left nav-global authenticated">
-      <%block name="navigation_global_links_authenticated">
-          <li class="nav-global-01">
-            <a href="${marketing_link('HOW_IT_WORKS')}">${_("How it Works")}</a>
-          </li>
-          <li class="nav-global-02">
-            <a href="${marketing_link('COURSES')}">${_("Find Courses")}</a>
-          </li>
-          <li class="nav-global-03">
-            <a href="${marketing_link('SCHOOLS')}">${_("Schools & Partners")}</a>
-          </li>
-      </%block>
-    </div>
+      % if not course or disable_courseware_header:
+    <nav aria-label="Main" class="nav-main">
+      <ul class="left nav-global authenticated">
+        <%block name="navigation_global_links_authenticated">
+            <li class="nav-global-01">
```

Comment:
NIT: It looks like this was already being done in this file, but it seemed like a number of places we are using classes instead of IDs here ("nav-global-n", "nav-courseware-n", etc.)